.drone.star

```diff
@@ -71,7 +71,6 @@ config = {
             "suites": {
                 "oC10Basic1": [
                     "webUIAccount",
-                    "webUIWebdavLocks",
```

Comment:
This pipeline has too few tests. Please add other suite to balance out the time.

Long pipelines:
```
SharingAccept
Files5
SharingPublic2
```